lbrynet/core/BlobMirror.py

```diff
@@ -0,0 +1,57 @@
+from random import choice
+import logging
+
+from twisted.internet import defer
+import treq
+
+from lbrynet import conf
+
+
+log = logging.getLogger(__name__)
+
+
+class BlobMirror(object):
+    def __init__(self, blob_manager, servers=None, client=None):
+        self.servers = list(servers or conf.settings['blob_server_domains'])
```

Comment:
id prefer to require that servers be passed in. then we can avoid a dependency on conf, which is already all over our code